### DISCUSSION TITLE
Disable Route Interception for nested routes

### DISCUSSION THREAD

**Discussion by phibr0** [vercel/next.js#59433] [App Router]
I am currently working on a Next.js project where I have implemented a modal that opens when navigating to a specific route (e.g., `/[id]/`). This functionality is working as expected.
However, I have encountered a challenge when trying to open a different page within the same modal. While I have managed to achieve this, I am facing an issue when the user is on the standard page (the non-modal route) and clicks a link to open the subpage. In this scenario, the subpage is intercepted and opens in a modal, which is not the desired behavior.
To illustrate, here is my route structure:

![Screenshot 2023-12-09 at 09 34 02](https://github.com/vercel/next.js/assets/59741989/294b065a-264e-435b-8f9b-fea9bcbc1f64)

I am looking for a way to disable route interception for this specific case, so that when the user is on the standard page and clicks the link to the subpage, it opens as a standard page rather than in a modal.

[![Edit phibr0/next-intercepted-modal/main](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/github/phibr0/next-intercepted-modal/main?file=%2Fapp%2Fpage.tsx&embed=1)

**Comment by kenaku** [comment]
Did somebody came up with a solution to that problem? Pattern seems pretty common.

It would be great to have an ability to bail from intercepting routes, like a Link prop or something...

**Reply by phibr0** [reply]
we didn't find a proper solution and are currently using hard navigations using regular anchor tags

**Reply by DEBargha2004** [reply]
can we just remove the subpage instead in the intercepted routes, which i think will solve as no route to be intercepted . To fix the not-found problem we can add a default.tsx file in the parallel route
